Sources/PrimerSDK/Classes/PCI/Services/Network/URLSessionStack.swift

```diff
@@ -325,7 +325,11 @@ internal extension URLSessionStack {
     private func url(for endpoint: Endpoint) -> URL? {
         guard let urlStr = endpoint.baseURL else { return nil }
         guard let baseUrl = URL(string: urlStr) else { return nil }
-        let url = baseUrl.appendingPathComponent(endpoint.path)
+        var url = baseUrl
+        
+        if endpoint.path != "" {
+            url = baseUrl.appendingPathComponent(endpoint.path)
```

Comment:
I'd expect some unit test for something as simple as this change @BorisNikolic

Reply:
Tests are merged.